doc/tutorials/make_your_first_quantum_circuit.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>quotation block renders funny, can make this all part of the same line:</p><p><br></p><p>Let's use the qubit simulator device provided by PennyLane. Devices are loaded in PennyLane via the function: <a href="https://pennylane.readthedocs.io/en/late/code/init.html#pennylane.device" target="_blank">pennylane.device( )</a>  </p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>